include/enclave/inside/ocalls.h

```diff
@@ -10,7 +10,12 @@ extern "C"
 {
     extern sgx_status_t SGX_CDECL ocallLogError(const char* msg);
 
+// In enclave release mode (i.e NDEBUG set) we disable debug logging
+#ifdef NDEBUG
+    void ocallLogDebug(const char* msg) { ; };
+#else
```

Comment:
Both methos need to have different return values to prevent the `RELEASE` version from doing any kind of OCall. Whereas the second signature is that corresponding to an OCall.